f600233d1b154d507b782f84604bcb12/update/sys_hub_action_type_definition_ae1e62bedb1a59108c045e8cd396190a.xml

```diff
@@ -159,10 +173,25 @@
   //var igce = null;
   var estCurrentFyObl = null;
   var igce = new Igce(inputs.acquisition_package);   
-  var dummyFunding = [
+  
+  let fundingDocId = null;
+  let fundingTypeCode = null;
+  if (inputs.funding_request_type === "MIPR") {
+    fundingDocId = inputs.mipr_number;
+    fundingTypeCode = "MIPR";
+  }
+  else if (inputs.funding_request_type === "FS_FORM") {
+    fundingDocId = inputs.ginv_order_number;
+    fundingTypeCode = "7600";
+  }
+  else {
+    throw "No Funding Number foud for this acquisition package";
+  }
+  
+  var fundingDoc = [
     {
-      fundingDocId: "DUMMYFORM9",
-      fundingType: "",
+      fundingDocId: fundingDocId,
+      fundingType: fundingTypeCode,
       fundingTypeCode: "",
```

Comment:
Requires correction as discussed via chat
![image](https://github.com/dod-ccpo/atat-snow/assets/77642966/fc2124c8-1ad9-4f7a-843a-22ed1098b1ec)

Reply:
Fixed in https://github.com/dod-ccpo/atat-snow/pull/400/commits/b8ef6f1eddd77bb5f63a7a55bb092dab2146aa96